mne/viz/ica.py

```diff
@@ -329,17 +356,50 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
 
     # calculations
     # ------------
+
     if isinstance(inst, BaseRaw):
+        # if reject is set, reject
+        if reject == 'auto':
+            reject = getattr(ica, 'reject_', None)
+        if reject is not None:
+            data = inst.get_data()
+            data, drop_inds = _reject_data_segments(data, ica.reject_,
+                                                    flat=None, decim=None,
+                                                    info=inst.info,
+                                                    tstep=2.0)
+            inst_rejected = RawArray(data, inst.info)
+        else:
+            inst_rejected = inst
+            drop_inds = None
         # break up continuous signal into segments
         from ..epochs import _segment_raw
+        inst_rejected = _segment_raw(inst_rejected, segment_length=2.,
+                                     verbose=False,
+                                     preload=True)
```

Comment:
don't they fit in a line? if not, put every parameter on its own line. But mixing looks weird.